api/app/services/demonstration/data_loader.rb

```diff
@@ -29,6 +29,11 @@ def import_projects
         next if File.file?(File.join(child, ".skip"))
         Importer::Project.new(child, cli_user, @logger).import(true)
       end
+      # Make all demo projects live
```

Comment:
I'd move this to Importer::Project and just have that service create non-draft projects by default. This block is problematic, too, because it's not just making all demo projects live—its making _all_ projects live. If we remove the part of the demo data loader that clears data before import, we'd have a problem.